src/anyio/_backends/_asyncio.py

```diff
@@ -1364,4 +1369,10 @@ def close(self) -> None:
             self._loop.close()
 
     def call(self, func: Callable[..., Awaitable], *args, **kwargs):
-        return self._loop.run_until_complete(func(*args, **kwargs))
+        retval = self._loop.run_until_complete(func(*args, **kwargs))
+        if len(self._exceptions) == 1:
+            raise self._exceptions[0]
+        elif self._exceptions:
+            raise ExceptionGroup(self._exceptions)
```

Comment:
If you raise this is a fixture teardown pytest will report the test result and this failure separately

Reply:
Isn't that what we want though?

Reply:
Yes but it's not what's in the changelog

> exceptions raised in synchronous callbacks will be raised then, essentially failing the test

pytest can report a test passing and then failing the teardown

Reply:
I'll update the changelog then.

There's another problem: since `self._loop.run_until_complete(...)` raises `StopAsyncIteration` on fixture teardown, it won't look at the accumulated exceptions. And even if it did, there's *another* problem: the exception list is not cleared between runs, so the same exceptions would be raised again!

Reply:
I've updated the changelog to match the actual behavior.